src/frame/content.rs

```diff
@@ -53,6 +53,8 @@ pub enum Content {
     Private(Private),
     /// A value containing the parsed contents of a table of contents frame (CTOC).
     TableOfContents(TableOfContents),
+    /// A value containing the parsed contents of a Unique File Identifier frame (UFID)
+    UFID(Ufid),
```

Comment:
Could you please rename the UFID enum variant and Ufid struct to `UniqueFileIdentifier`? Other frame types also use the written out name, so that would be consistent.